src/Models/StoredEvent.php

```diff
@@ -38,9 +38,9 @@ public static function createForEvent(ShouldBeStored $event, string $uuid = null
         return $storedEvent;
     }
 
-    public function getEventClassAttribute(string $value): string
+    public static function getEventClassAttribute(string $value): string
```

Comment:
I'm thinking this one doesn't need to be static.

Reply:
Done!